app/lib/features/onboarding/pages/sign_up_page.dart

```diff
@@ -52,156 +51,150 @@ class _SignupPageState extends ConsumerState<SignupPage> {
-            SliverAppBar(
-              leading: desktop
-                  ? InkWell(
-                      onTap: () => context.goNamed(Routes.main.name),
-                      child: const Icon(Atlas.arrow_left_circle),
-                    )
-                  : null,
-            ),
-            SliverToBoxAdapter(
-              child: Form(
-                key: formKey,
-                child: Column(
-                  crossAxisAlignment: CrossAxisAlignment.center,
-                  children: [
-                    const SizedBox(height: 80),
-                    SizedBox(
-                      height: 50,
-                      width: 50,
-                      child: SvgPicture.asset('assets/icon/acter.svg'),
-                    ),
-                    const SizedBox(height: 40),
-                    Text(
-                      AppLocalizations.of(context)!.onboardText,
-                    ),
-                    const SizedBox(height: 10),
-                    Text(
-                      AppLocalizations.of(context)!.createAccountText,
-                    ),
-                    const SizedBox(height: 20),
-                    SignUpTextField(
-                      hintText: AppLocalizations.of(context)!.name,
-                      controller: name,
-                      validatorText: AppLocalizations.of(context)!.missingName,
-                      type: SignUpOnboardingTextFieldEnum.name,
-                    ),
-                    SignUpTextField(
-                      hintText: AppLocalizations.of(context)!.username,
-                      controller: username,
-                      validatorText:
-                          AppLocalizations.of(context)!.emptyUsername,
-                      type: SignUpOnboardingTextFieldEnum.userName,
-                    ),
-                    SignUpTextField(
-                      hintText: AppLocalizations.of(context)!.password,
-                      controller: password,
-                      validatorText:
-                          AppLocalizations.of(context)!.emptyPassword,
-                      type: SignUpOnboardingTextFieldEnum.password,
-                    ),
-                    SignUpTextField(
-                      hintText: AppLocalizations.of(context)!.token,
-                      controller: token,
-                      validatorText: AppLocalizations.of(context)!.emptyToken,
-                      type: SignUpOnboardingTextFieldEnum.token,
-                    ),
-                    const SizedBox(height: 30),
-                    Container(
-                      margin: const EdgeInsets.symmetric(horizontal: 20),
-                      child: RichText(
-                        textAlign: TextAlign.start,
-                        text: TextSpan(
-                          // Note: Styles for TextSpans must be explicitly defined.
-                          // Child text spans will inherit styles from parent
+    final List<Widget> actions = [];
```

Comment:
I don't think it's suitable to have this actions var in ```build()``` because it updates frequently and may reset the list, rather have it defined outside in class.